.mvn/extensions.xml

```diff
@@ -3,6 +3,6 @@
     <extension>
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>tycho-build</artifactId>
-        <version>2.7.0</version>
+        <version>3.0.0-SNAPSHOT</version>
```

Comment:
2.7.5 should be fine here as we do not need any snapshot features in the build extension here.